### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by bipoza**
**I have updated some configuration elements for a more comfortable installation.**

Thank you very much for the capacitor plugin. We needed it!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,33 +159,45 @@ Update the project:
...
 npx cap update ios
 ```
+Sync the project:
 
+```bash
+npx cap sync ios
+```
 Make the following changes to your `AppDelegate.swift` file:
+> To consider: Read the official documentation. The import method is different depending on the swift version. Capacitor version 1.4.0 uses Swift version 4.2. 
+[Check version](https://github.com/ionic-team/capacitor/blob/master/.swift-version)
```
Comment (mikejpeters) [inline README.md]: I don't understand. How does the import method change?

The Branch docs only show an example for Swift 4.2 but not Swift 4.0

File: README.md [hunk 2]
```diff
@@ -155,33 +159,45 @@ Update the project:
...
   func application(_ application: UIApplication, continue userActivity: NSUserActivity, restorationHandler: @escaping ([UIUserActivityRestoring]?) -> Void) -> Bool {
     // Called when the app was launched with an activity, including Universal Links.
     // Feel free to add additional processing here, but if you want the App API to support
     // tracking app url opens, make sure to keep this call
-+   Branch.getInstance()!.continue(userActivity)
++   Branch.getInstance().continue(userActivity)
     return CAPBridge.handleContinueActivity(userActivity, restorationHandler)
   }
+
+// Optional (You probably have to add the entire function)
```
Comment (mikejpeters) [inline README.md]:
```suggestion
+ // Branch push notification handler (optional)
```

File: README.md [hunk 3]
```diff
@@ -155,33 +159,45 @@ Update the project:
...
+  func application(_ application: UIApplication, didReceiveRemoteNotification userInfo: [AnyHashable : Any], fetchCompletionHandler completionHandler: @escaping (UIBackgroundFetchResult) -> Void) {
+  // handler for Push Notifications
++  Branch.getInstance().handlePushNotification(userInfo)
+}
```
Comment (mikejpeters) [inline README.md]: Because this entire function doesn't exist in the Capacitor iOS template, can you add `+ ` before each line?

File: README.md [hunk 4]
```diff
@@ -155,33 +159,45 @@ Update the project:
...
     // Feel free to add additional processing here, but if you want the App API to support
     // tracking app url opens, make sure to keep this call
-+   Branch.getInstance()!.continue(userActivity)
++   Branch.getInstance().continue(userActivity)
     return CAPBridge.handleContinueActivity(userActivity, restorationHandler)
   }
+
+// Optional (You probably have to add the entire function)
+  func application(_ application: UIApplication, didReceiveRemoteNotification userInfo: [AnyHashable : Any], fetchCompletionHandler completionHandler: @escaping (UIBackgroundFetchResult) -> Void) {
+  // handler for Push Notifications
```
Comment (mikejpeters) [inline README.md]:
```suggestion

```

File: README.md (hunk 1 above)
Comment (bipoza) [reply README.md]: I don't think it's necessary to document that, you're right. Currently capacitor uses version 4.2, therefore I do not think this contributes anything.
alt text

**Comment by tafelnl** [comment]
@bipoza

Good news: my PR (https://github.com/BranchMetrics/capacitor-branch-deep-links/pull/22) has been merged. I think that make this PR obsolete as well right? In that case, this one can be closed as well @echo-branch

**Comment by bipoza** [comment]
Of course, very good news 🥳
Thanks for your contribution!

**Pull request closed without merging**